.secrets.baseline

```diff
@@ -156,7 +156,7 @@
         "filename": "contributions/catalog/Dockerfile",
         "hashed_secret": "b6ee60926c0a426addcbb7e087d4274498f35b1c",
         "is_verified": false,
-        "line_number": 22,
+        "line_number": 20,
```

Comment:
@sandeep-ps Unsure if on which secrets were changed because of a port change. GITHUB_CLIENT_SECRET is still unchanged in the catalog/Dockerfile

Reply:
No, so this is part of detect secrets tool. It's just saying that the catalog/Dockerfile got updated and the line number of a non-secret in that file has changed.